telemetry/specs/INT.mdk

```diff
@@ -1009,24 +1006,28 @@ each bit corresponds to a specific standard metadata as specified in Section 3.
   Details of the metadata semantics YANG model can be accessed at the link below:
   https://github.com/p4lang/p4-applications/blob/master/telemetry/code/models/p4-dtel-metadata-semantics.yang
   
-  Bits 0 - 13 are Baseline INT Instructions and Bit 14, Domain Specific Instruction.
-  Domain Specific Instruction is an instruction that requires additional 
-  processing of Domain Specific ID, Domain Specific Flags (DS Flags) and 
-  Domain Specific Field (DS Field). If the Domain Specific ID doesn't match the 
-  Domain ID of the node, then the transit node is required to pad the node's 
-  INT Metadata stack with the special all-ones reserved value for Domain Specific 
+  Bits 0 - 14 are Baseline INT Instructions.
+  
+  * Domain Specific ID (16b): the unique ID of the INT Domain.
+  
+  * DS Instruction (16b): Instruction bit map specific to the INT domain identified by the 
+  Domain Specific ID. Domain Specific Instruction is an instruction that requires additional 
+  processing of Domain Specific Flags (DS Flags) for the INT Domain identified by Domain Specific ID. 
+  If the Domain Specific ID doesn't match the Domain ID of the node, then the transit node is 
+  required to pad the node's INT Metadata stack with the special all-ones reserved value for Domain Specific 
   Metadata length as calculated by subtracting the Baseline Metadata Length from Hop_ML.
```

Comment:
I think the text about Domain Specific ID that is not known to the node should go after the text about Domain Specific ID that is known to the node.

More importantly, I want to make it clear that when the Domain Specific ID is not known, as an alternative to padding with all 1s, the node could not add any metadata at all. That is why I suggested the following text:

If the Domain Specific ID does not match any Domain ID known to this node, then
the node is required to either:

- Pad the node's INT Metadata stack with the special all-ones reserved value for a Domain Specific Metadata length, calculated by subtracting from the Hop ML a length computed from all bits in the 16-bit INT Instruction field, or

- Skip INT processing altogether and not insert any metadata into the packet

Reply:
Done.